pkg/config/shared.go

```diff
@@ -17,14 +17,20 @@ package config
 import (
 	"context"
 	"crypto/tls"
+	"io/ioutil"
 	"time"
 
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	ttnblob "go.thethings.network/lorawan-stack/pkg/blob"
```

Comment:
I see a 👍 but what is the action here?

Reply:
I think the whole `shared.go` is a terrible idea, however we need to avoid import cycles and be consistent with what we already have.
Refactoring and changing this is out of scope of this PR.